guides/common/modules/con_synchronizing-content-between-servers.adoc

```diff
@@ -19,7 +19,7 @@ Configure your {Project} to synchronize content by using export and import.
 For more information, see xref:configuring-projectserver-to-synchronize-content-by-using-exports_{context}[].
 
 .Additional resources
-* For more information on ISS use cases and scenarios, see xref:iss-scenarios_{context}[].
+* For more information on ISS use cases and scenarios, see {PlanningDocURL}iss-scenarios_{context}[{ISS} scenarios] in _{PlanningDocTitle}_.
```

Comment:
I believe that we have to resolve "{context}" here; it should be `planning`: https://github.com/theforeman/foreman-documentation/blob/master/guides/doc-Planning_for_Project/master.adoc?plain=1#L3